src/uct/ib/rc/verbs/rc_verbs_iface.c

```diff
@@ -44,24 +44,19 @@ static ucs_config_field_t uct_rc_verbs_iface_config_table[] = {
 static void uct_rc_verbs_handle_failure(uct_ib_iface_t *ib_iface, void *arg,
                                         ucs_status_t status)
 {
-    struct ibv_wc     *wc      = arg;
     uct_rc_iface_t    *iface   = ucs_derived_of(ib_iface, uct_rc_iface_t);
     ucs_log_level_t    log_lvl = UCS_LOG_LEVEL_FATAL;
+    struct ibv_wc     *wc;
     uct_rc_verbs_ep_t *ep;
 
+    wc = arg;
```

Comment:
why this change?

Reply:
left overs from previous implementation, will revert